test-src/container_test.js

```diff
@@ -4,10 +4,8 @@ let uuid = require('uuid');
 
 describe('Azure Blob Container', () => {
   let containerName = uuid.v4();
-  let accountName = 'accountNameTest';
-
-  let accountKeyString = new Buffer('JavaScript');
-  let accountKey = accountKeyString.toString('base64');
+  let accountName = process.env.AZURE_ACCOUNT_NAME;
+  let accountKey = process.env.AZURE_ACCOUNT_KEY;
```

Comment:
great!  In other taskcluster projects, we've also used `typed-env-config` to do the environment variable and `taskcluster-lib-loader` to set up the configuration, but I think that for now we should stick to simple environment variables!  I'm still working on getting credentials to you to run the tests.